### ISSUE TITLE
[Bug] / Question How to remove space after lora?

### ISSUE THREAD

**Issue by thijsi123** [Physton/sd-webui-prompt-all-in-one#294]
### Issue Feedback

- [X] I confirm that I have searched for a solution to this issue in the [FAQ](https://aiodoc.physton.com/FAQ.html) and couldn't find a solution.
- [X] I confirm that I have searched for this issue in the [Issues](https://github.com/Physton/sd-webui-prompt-all-in-one/issues) list (including closed ones) and couldn't find a solution.
- [X] I confirm that I have read the [Wiki](https://aiodoc.physton.com/) and couldn't find a solution.

### Describe the Issue

So, after changing some settings (see attached image) I still can't seem to find an option to disable extra space after LoRA.
![image](https://github.com/Physton/sd-webui-prompt-all-in-one/assets/64159148/3e28266c-550c-43b6-80ba-85cd8e261417)

The problem is that it adds a space after > i.e. 
<lora:BADAPPLE:1> Bad apple style,
becomes                
<lora:BADAPPLE:1>  Bad apple style,

so when pressing the LoRA again it only removes '<lora:BADAPPLE:1>' instead '<lora:BADAPPLE:1>  Bad apple style,' because of the extra space.

### Steps to Reproduce

click on lora
click on lora to remove the activation text and lora
not everything is gone

### Screenshot or log

![badapple](https://github.com/Physton/sd-webui-prompt-all-in-one/assets/64159148/1e758b4f-7380-4dae-872b-0e01c2a0690a)

### OS

Windows

### Browser

Edge

### Stable Diffusion WebUI version

v1.7.0

### Extension version

658ae822

### Python version

3.10.0

### Gradio version

3.41.2

**Comment by thijsi123** [comment]
There's also a problem that after typing more tags after the lore it adds an extra space widening the gap.

**Comment by thijsi123** [comment]
I fixed my issue by changing
(Hn=this.autoRemoveSpace?"":" ")
to
(Hn=this.autoRemoveSpace?"":"")
inside 
extensions\sd-webui-prompt-all-in-one\javascript\main.entry.js

may I request a toggle for this for future updates?

**Comment by Physton** [comment]
![image](https://github.com/Physton/sd-webui-prompt-all-in-one/assets/12066768/077fa740-7e86-4d13-afe4-0c1597fb6eaf)
Please check `Whether to remove the spaces after each comma in the keyword.`

**Comment by thijsi123** [comment]
That removes every space which causes the issue again.

**Comment by Lalimec** [comment]
yes this is an annoying feature. Especially when i use loras with dynamic prompt. Try {<lora:aaa:1>|<lora:bbb:1>|<lora:ccc:1>|<lora:ddd:1>} and enjoy the show lol.
